pkg/util/helper/workstatus.go

```diff
@@ -36,10 +38,9 @@ const (
 
 // AggregateResourceBindingWorkStatus will collect all work statuses with current ResourceBinding objects,
 // then aggregate status info to current ResourceBinding status.
-func AggregateResourceBindingWorkStatus(c client.Client, binding *workv1alpha2.ResourceBinding, workload *unstructured.Unstructured) error {
+func AggregateResourceBindingWorkStatus(c client.Client, binding *workv1alpha2.ResourceBinding, workload *unstructured.Unstructured, eventRecorder record.EventRecorder) error {
 	workList, err := GetWorksByBindingNamespaceName(c, binding.Namespace, binding.Name)
 	if err != nil {
-		klog.Errorf("Failed to get works by ResourceBinding(%s/%s): %v", binding.Namespace, binding.Name, err)
```

Comment:
Is there any specific reason you removed this log?
I suppose a whole failure path would help debug.

Reply:
This log is duplicated because the return err will be printed by `syncBinding`. BTW, other errors are not printed in `AggregateResourceBindingWorkStatus`.

Reply:
In my experience, sometimes when I want to trace an error log, I want to find the underlying function which first raised it, it will be difficult, especially in many layers. 

Never mind. no big deal.